### PR TITLE
chore(comments): Drop dead code from comments app

### PR DESCRIPTION
**Pull request by susnux**
## Summary
Drop code that is not used anymore from repository - also drop "comments" script / entry point which is unused.
## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by emoral435 — APPROVED** [review]

File: apps/comments/.l10nignore [hunk 1]
```diff
@@ -1,3 +0,0 @@
-# compiled vue templates
-src/templates.js
-js/
```
Comment (artonge) [inline apps/comments/.l10nignore:3]: Isn't that line needed?

**Review by artonge — APPROVED** [review]
What was the use of this script?

File: apps/comments/.l10nignore (hunk 1 above)
Comment (susnux) [reply apps/comments/.l10nignore:3]: `src/templates.js` is removed
`js/` does not exist as compiled code is provided in `../../dist`

Comment (artonge) [reply apps/comments/.l10nignore:3]: Sure :see_no_evil:

**Comment by susnux** [comment]
Drone (samba) unrelated

**Comment by susnux** [comment]
> What was the use of this script?

Old API that was dropped long time ago it seems. The only API used by other apps is the `OCA.Comments.View` used by e.g. announcement center

**Comment by artonge** [comment]
I remember @nickvergessen was also afraid to lose an API for Talk

**Comment by nickvergessen** [comment]
> Old API that was dropped long time ago it seems

It was not dropped. I prevent removal just last month....

This is public API:
https://github.com/nextcloud/server/blob/aa5f037af71c915424c6dcfd5ad2dc82797dc0d6/lib/private/Comments/Manager.php#L1595-L1603
Can we just keep it for now?

We can deprecate and drop in 3 years if there are reasons apart from "just because"

**Comment by nickvergessen** [comment]
See https://github.com/nextcloud/server/pull/41491#pullrequestreview-1732075803 and further

**Comment by nickvergessen** [comment]
Reverting in https://github.com/nextcloud/server/pull/42207 to restore the nice functionality and public API

**Comment by susnux** [comment]
@nickvergessen that is a different script that is still provided. `comments-app` is still provided.

**Pull request merged**